lib/src/ui/codelessly_widget.dart

```diff
@@ -567,3 +616,31 @@ class _NavigationBuilderState extends State<_NavigationBuilder> {
   @override
   Widget build(BuildContext context) => widget.builder(context);
 }
+
+/// Retrieves a canvas ID for a layout group based on the current screen size.
+/// Returns null if the layout group does not have a canvas for the current
+/// screen size or if the layout group does not exist.
+String? getCanvasIDForLayoutGroup(
```

Comment:
Make it private and static?

Reply:
Private..yes
Static...its global level!